Index: api/setup.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>firebaseConfig = {\r\n  \"apiKey\": \"AIzaSyAP-ytff2KHx1EkKDjnncLu2EWx5SL2EPc\",\r\n  \"authDomain\": \"zeitask.firebaseapp.com\",\r\n  \"projectId\": \"zeitask\",\r\n  \"storageBucket\": \"zeitask.appspot.com\",\r\n  \"messagingSenderId\": \"708185522867\",\r\n  \"appId\": \"1:708185522867:web:4d0cde2fe490945e2ee6c6\",\r\n  \"measurementId\": \"G-4LC53J2RGY\"\r\n}
===================================================================
diff --git a/api/setup.py b/api/setup.py
--- a/api/setup.py	
+++ b/api/setup.py	
@@ -1,3 +1,5 @@
+import json
+
 firebaseConfig = {
   "apiKey": "AIzaSyAP-ytff2KHx1EkKDjnncLu2EWx5SL2EPc",
   "authDomain": "zeitask.firebaseapp.com",
@@ -6,4 +8,9 @@
   "messagingSenderId": "708185522867",
   "appId": "1:708185522867:web:4d0cde2fe490945e2ee6c6",
   "measurementId": "G-4LC53J2RGY"
-}
\ No newline at end of file
+}
+
+
+def returns_json(value):
+    return json.dumps(value)
+
